Description: Upstream changes introduced in version 0.3-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 librest-utils-perl (0.3-1) UNRELEASED; urgency=low
 .
   * New upstream version.
   * Changed source format to 3.0 (quilt)
 .
 The person named in the Author field signed this changelog entry.
Author: Jaldhar H. Vyas <jaldhar@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- librest-utils-perl-0.3.orig/MANIFEST
+++ librest-utils-perl-0.3/MANIFEST
@@ -12,4 +12,3 @@ t/get-body.t
 t/http-tunnel.t
 Makefile.PL
 META.yml
-SIGNATURE    Added here by Module::Build
